hubblestack/config.py

```diff
@@ -0,0 +1,1995 @@
+# -*- coding: utf-8 -*-
+"""
+All Hubble configuration loading and defaults should be in this module
+"""
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals, generators
```

Comment:
I think we can remove then __future__ imports we're running python 3, do we still need them for some reason?

Reply:
Excellent point... I'll have to go fix that.